privaterelay/templates/includes/login.html

```diff
@@ -6,8 +6,15 @@
 {% block content %}
 
 <div class="user-area">
-	<firefox-apps></firefox-apps>
 	{% if request.user.is_anonymous %}
+	<a
+	 data-entrypoint="relay-sign-in-header"
+	 data-form_type="button"
+	 data-ga="send-ga-funnel-pings"
+	 data-event-category="Sign In Buttons"
+	 data-event-label="sign-in-btn-header"
+	 class="sign-in sign-up button"
```

Comment:
Suggestion: Use the `.header-link` class to make the button behave like the Home/FAQ buttons. You may need to play with the padding to compensate. 

```suggestion
	 class="header-link"
```

<img width="451" alt="image" src="https://user-images.githubusercontent.com/2692333/128419135-c2e4c1ac-a83c-48fc-b64e-30e0222d4e1c.png">